controller-server/src/main/java/com/yahoo/vespa/hosted/controller/role/PathGroup.java

```diff
@@ -26,33 +28,44 @@
              "/nodes/v2/{*}",
              "/orchestrator/v1/{*}"),
 
-    /** Paths used when onboarding and creating a new tenants */
+    /** Paths used for creating user tenants */
     onboardingUser("/application/v4/user"),
 
-    // Tenant parameter is ignored here as context for the role is not defined until after a tenant has been created
+    /** Paths used for creating tenants with access control */
     onboardingTenant("/application/v4/tenant/{ignored}"),
 
     /** Read-only paths used when onboarding tenants */
     onboardingTenantInformation("/athenz/v1/",
                                 "/athenz/v1/domains"),
 
+    /** Paths used for user management */
+    userManagement("/user/v1/{*}"), // TODO probably add tenant and application levels.
 
-    /** Paths used by tenant/application administrators */
-    tenant("/application/v4/",
-           "/application/v4/property/",
-           "/application/v4/tenant/",
-           "/application/v4/tenant-pipeline/",
+    /** Paths used by tenant administrators */
+    tenantInfo("/application/v4/",
+               "/application/v4/property/",
+               "/application/v4/tenant/",
+               "/application/v4/tenant-pipeline/"),
+
+    /** Paths used by tenant administrators */
+    tenant(Matcher.tenant,
```

Comment:
Good, I was thinking of something like this, e.g. that a path group had to declare if a limited context was required or not. But this is better 👍